### ISSUE TITLE
[BUG] Get-PnPTenantSite: Missing description

### ISSUE THREAD

**Issue by Swampen** [pnp/powershell#180]
### Reporting an Issue or Missing Feature
`Get-PnPTenantSite -Url <site>` returns an object without the `Description` property.
Using `Get-PnPTenantSite | Format-List`, shows the description property

### Expected behavior 
```powershell
Get-PnPTenantSite -Url <site> | Select-Object Description
```
Outputs the description of the site

### Actual behavior 
Returns null

### Steps to reproduce behavior
```powershell
Get-PnPTenantSite -Url <site> | Select-Object Description
```

### What is the version of the Cmdlet module you are running?
1.2.4-nightly

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by ToddKlindt** [comment]
Neither the PnP.PowerShell or the Microsoft.Online.SharePoint.PowerShell get site cmdlets (Get-PnPSite, Get-PnPTenantSite, or  Get-SPOSite) show a property for Description or SiteDescription, as verified by Get-Member. I wonder if it's a limitation of the underlying API.

**Comment by Swampen** [comment]
Microsoft.Online.SharePoint.PowerShell exposes this as shown in the screenshot
![image](https://user-images.githubusercontent.com/32768682/105900150-f07c6600-601b-11eb-890e-7ac1efb45bfe.png)

**Comment by ToddKlindt** [comment]
You say it's in the Microsoft.Online.SharePoint.PowerShell module but your screenshot is of Get-PnPTenantSite, which is in the PnP.PowerShell module. Regardless, I can't replicate finding a Description Property. Here's what I did:
![image](https://user-images.githubusercontent.com/2659747/105910405-84cfd400-5fee-11eb-9291-4f2ead0c5ee7.png)
Am I doing something differently than you are?

**Comment by erwinvanhunen** [comment]
actually @Swampen  is using an older build where we still returned the plain object returned from the server. However, we added some properties to stay in sync with the SPO Management Shell, and as a result we're returning a different type of object. The description property was not present on that new object, but I just added it and it will be available in 1.2.6-nightly.

**Comment by ToddKlindt** [comment]
> actually @Swampen is using an older build where we still returned the plain object returned from the server. However, we added some properties to stay in sync with the SPO Management Shell, and as a result we're returning a different type of object. The description property was not present on that new object, but I just added it and it will be available in 1.2.6-nightly.

I thought that might be the case so I included my versions. I'm glad you figured it out.
